storage/backend/backend.go

```diff
@@ -62,16 +62,20 @@ type Snapshot interface {
 }
 
 type backend struct {
+	// size and commits are used with atomic operations so they must be
+	// 64-bit aligned, otherwise 32-bit tests will crash
+
+	// size is the number of bytes in the backend
+	size int64
```

Comment:
@xiang90 I don't think so; the accessor methods don't bother locking, they just do an atomic read.